_security-analytics/threat-intelligence/api/findings.md

```diff
@@ -0,0 +1,267 @@
+}
+```
+
+### Response fields
+
+Threat intelligence alerts can have one of the following states.
+
+| State  | Description  |
+| :---- | :--- |
+| `ACTIVE`   | The alert is ongoing and unacknowledged. Alerts remain in this state until they are acknowledged, the trigger associated with the alert is deleted, or the threat intelligence monitor is deleted entirely. |
+| `ACKNOWLEDGED` | The alert is acknowledged but the root cause of the alert has not been addressed.  |
+| `COMPLETED` | The alert is no longer ongoing. Alerts enter this state after the corresponding trigger evaluates to `false`.   |
+| `DELETED` | The monitor or trigger for the alert was deleted while the alert was active.  |
+
+---
+
+## Update alerts status API 
+
+Updates the status of the specified alerts to  `ACKNOWLEDGED` or `COMPLETED`. Only alerts in the `ACTIVE` state can be updated. 
+
+## Path and HTTP methods
+
+```json
+PUT /plugins/security_analytics/threat_intel/alerts/status
+```
+
+### Example requests
+
+The following example updates status of the specified alerts to `ACKNOWLEDGED`:
+
+```json
+PUT /plugins/security_analytics/threat_intel/alerts/status?state=ACKNOWLEDGED&alert_ids=<alert-id>,<alert-id>
+```
+
+The following example updates status of the specified alerts to `COMPLETED`:
+
+```json
+PUT /plugins/security_analytics/threat_intel/alerts/status?state=COMPLETED&alert_ids=alert_ids=<alert-id>,<alert-id>
+```
+
+### Example response
+
+```json
+{
+  "updated_alerts": [
+    {
+      "id": "906669ee-56e8-4f40-a12f-ab4c274d7521",
+      "version": 1,
+      "schema_version": 0,
+      "seq_no": 2,
+      "primary_term": 1,
+      "trigger_id": "regwarg",
+      "trigger_name": "regwarg",
+      "state": "ACKNOWLEDGED",
+      "error_message": null,
+      "ioc_value": "example-has00001",
+      "ioc_type": "hashes",
+      "severity": "high",
+      "finding_ids": [
+        "a9c10094-6139-42b3-81a8-867dffbe381d"
+      ],
+      "acknowledged_time": 1722039091209,
+      "last_updated_time": 1722039091209,
+      "start_time": 1722038395105,
+      "end_time": null
+    },
+    {
+      "id": "56e8-4f40-a12f-ab4c274d7521-906669ee",
+      "version": 1,
+      "schema_version": 0,
+      "seq_no": 2,
+      "primary_term": 1,
+      "trigger_id": "regwarg",
+      "trigger_name": "regwarg",
+      "state": "ACKNOWLEDGED",
+      "error_message": null,
+      "ioc_value": "example-has00001",
+      "ioc_type": "hashes",
+      "severity": "high",
+      "finding_ids": [
+        "a9c10094-6139-42b3-81a8-867dffbe381d"
+      ],
+      "acknowledged_time": 1722039091209,
+      "last_updated_time": 1722039091209,
+      "start_time": 1722038395105,
+      "end_time": null
+    }
+  ],
+  "failure_messages": []
+}
+```
+
+
+
+---
+
+## Get findings
+
+Returns threat intelligence Indicators of compromise (IOCs) findings. When the threat intelligence monitor finds a malicious IoC during a data scan, a finding is automatically added to the threat intelligence feed.
```

Comment:
It can simply be reworded as `When the threat intelligence monitor finds a malicious IoC during a data scan, a finding for that ioc is generated and accessible via the findings API` ? @Naarcha-AWS

Reply:
Sure. Also, AWS and Azure both use IOC, which is why its formatted with all caps versus IoC. See https://www.microsoft.com/en-us/security/business/security-101/what-are-indicators-of-compromise-ioc.

Reply:
@Naarcha-AWS Oh what I meant was that both IOC and IoC is used in this paragraph.